MediaBrowser.Plugins.SoundCloud/ImageProcessing/SoundCloudOverlayEnhancer.cs

```diff
@@ -121,23 +121,24 @@ protected async Task<MagickWand> EnhanceImageAsyncInternal(IHasImages item, Magi
                 catch (Exception exception1)
                 {
                     Exception exception = exception1;
-                    this.Logger.ErrorException("Error loading overlay: {0}", exception, overlayType);
+                    _logger.LogError(exception, "Error loading overlay: {0}", overlayType);
                 }
             }
             else
             {
-                this.Logger.Warn(string.Concat("SoundCloud - Undefined overlay type: ", overlayType), new object[0]);
+                // TODO wtf is this, can't it be formatted in a normal way?
+                _logger.LogWarning(string.Concat("SoundCloud - Undefined overlay type: ", overlayType), new object[0]);
```

Comment:
Shouldn't this just be
```suggestion
                _logger.LogWarning("SoundCloud - Undefined overlay type: {OverlayType}", overlayType);
```

Reply:
thanks :)